Error: The `form` does not need to be labelled by the `legend` as the `fieldset` & `legend` combination already provides those semantics.
Error: The `form` element was not requested.
diff --git a/output/Radio Group/4/31c79aab-82ab-4466-8310-30983a7ace00.html b/output/Radio Group/4/31c79aab-82ab-4466-8310-30983a7ace00.html
index ff262be..5315630 100644
--- a/output/Radio Group/4/31c79aab-82ab-4466-8310-30983a7ace00.html	
+++ b/output/Radio Group/4/31c79aab-82ab-4466-8310-30983a7ace00.html	
@@ -1,17 +1,15 @@
-<form aria-labelledby="theme-chooser-label">
-  <fieldset>
-    <legend id="theme-chooser-label">Choose a theme:</legend>
-    <div>
-      <input type="radio" id="light" name="theme" value="light">
-      <label for="light">Light</label>
-    </div>
-    <div>
-      <input type="radio" id="dark" name="theme" value="dark">
-      <label for="dark">Dark</label>
-    </div>
-    <div>
-      <input type="radio" id="high-contrast" name="theme" value="high-contrast">
-      <label for="high-contrast">High Contrast</label>
-    </div>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>Choose a theme:</legend>
+  <div>
+    <input type="radio" id="light" name="theme" value="light">
+    <label for="light">Light</label>
+  </div>
+  <div>
+    <input type="radio" id="dark" name="theme" value="dark">
+    <label for="dark">Dark</label>
+  </div>
+  <div>
+    <input type="radio" id="high-contrast" name="theme" value="high-contrast">
+    <label for="high-contrast">High Contrast</label>
+  </div>
+</fieldset>
\ No newline at end of file
